### ISSUE TITLE
Linkintegrity-Relations not created when creating new objects

### ISSUE THREAD

**Issue by pbauer** [plone/plone.app.linkintegrity#21]
Steps to reproduce:

1. create page1
2. create page2 and (during creation) add a link in the richtext-field to page1
3. delete page1: no LinkIntegrityNotificationException will be fired

If you edit page2 again (before deleting page1) a LinkIntegrityNotificationException will be fired

So far we tried replacing the subscriber with the following:
```xml
<subscriber
    for="Products.CMFCore.interfaces._content.IContentish
         zope.lifecycleevent.interfaces.IObjectAddedEvent"
    handler="..handlers.addedDexterity" />
```
```python
def addedDexterity(obj, event):
    from plone.app.referenceablebehavior.referenceable import IReferenceable as IRef
    if IRef.providedBy(obj):
        modifiedDexterity(obj, None)
```
Upon inspecting page1 after step 2. we can see that it has no backrelation:
```python
>>> from Products.Archetypes.interfaces.referenceable import IReferenceable
>>> IReferenceable(obj).getBackReferences()
[None]
```
it should be:
```python
>>> IReferenceable(obj).getBackReferences()
[<Document at /Plone/page2>]
```
We will probably have more time investigate further later today.

**Comment by rodfersou** [comment]
@pbauer I find that the update view at 1.5.5 version in pypi is not equal the one int the branch 1.5.x .. it just looks for archetype types, don't call this method https://github.com/plone/plone.app.linkintegrity/blob/1.5.x/plone/app/linkintegrity/browser/update.py#L11

Take a look into omelete.

**Comment by pbauer** [comment]
Fixed in 3.0
